gradle/missing-javadoc.gradle

```diff
@@ -141,6 +141,7 @@ configure([
   project(":plugins:mapper-annotated-text"),
   project(":plugins:mapper-murmur3"),
   project(":plugins:mapper-size"),
+  project(":plugins:query-insights"),
```

Comment:
Do you need to request this exclusion? For new plugins, we should probably just add JavaDoc as we go.

Reply:
Added javadoc coverage.

Reply:
Please remove the exclusion for the plugin, then.